common/lib/xmodule/xmodule/js/src/video/035_video_accessible_menu.js

```diff
@@ -296,11 +320,17 @@ function () {
     // ***************************************************************
 
     function changeFileType(event) {
-        var fileType = $(event.currentTarget).data('value');
+        var fileType = $(event.currentTarget).data('value'),
+            button = $('.download-link'),
+            wrapper = $('.video-download-button');
 
         this.videoAccessibleMenu.setValue(fileType);
         this.saveState(true, {'transcript_download_format': fileType});
         this.storage.setItem('transcript_download_format', fileType);
+
+        wrapper.find(button)
+            .text(interpolate(gettext('Download transcript (%(fileType)s)?'), { fileType: fileType }, true))
```

Comment:
Nit: this same message is constructed in two places. Having a helper function (passing in fileType) would be safer as it would ensure that the message is always consistent. Otherwise, it might be changed in one place but not the other.